build-mingw-w64.sh

```diff
@@ -124,3 +124,10 @@ for arch in $ARCHS; do
     cd ..
 done
 cd ..
+
+for arch in $ARCHS; do
+    copying_dir="$PREFIX/$arch-w64-mingw32/share/mingw32"
+    install -Dm644 COPYING "$copying_dir/COPYING"
+    install -Dm644 COPYING.MinGW-w64/COPYING.MinGW-w64.txt "$copying_dir/COPYING.MinGW-w64.txt"
+    install -Dm644 COPYING.MinGW-w64-runtime/COPYING.MinGW-w64-runtime.txt "$copying_dir/COPYING.MinGW-w64-runtime.txt"
```

Comment:
Can you avoid duplicating the file names here? Maybe a loop over the input names, and using `$(basename $src)` as the output filename? Given the slightly long and nontrivial names, it feels like something where one easily would add a typo on the right hand side...